src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -309,11 +313,35 @@ const AddOrEditExperienceForm = ({
           />
         </Grid>
       )}
-      <Grid item xs={12} align="end">
+      <Grid
+        item
+        xs={12}
+        style={{display: 'flex'}}
+        className={
+          onDelete ? classes.alignButtonsBetween : classes.alignButtonsEnd
+        }
```

Comment:
It's weird to have part of the style declared inline here. In general, I'd just add `flex` to both of these classes, but I don't think you actually need them if you add an empty element below

Reply:
inline style is removed.